docs/protocol/extension_statistics.md

```diff
@@ -217,4 +296,33 @@ message InferStatistics
   // should include the time to copy output tensor data from the GPU.
   StatisticDuration compute_output = 6;
 }
+
+// Inference batch statistics.
+message InferBatchStatistics
+{
+  // The size of the batch.
+  uint64 batch_size = 1;
+
+  // The number of times the batch size was executed on the
+  // model. A single model execution performs inferencing for the entire
+  // request batch and can perform inferencing for multiple requests if
+  // dynamic batching is enabled.
+  uint64 count = 2;
+
+  // The count and cumulative duration to prepare input tensor data as
```

Comment:
Should we have another message holding "StatisticDuration" messages, and use that in InferBatchStatistics and InferStatistics? or is it too much nesting?

Reply:
Yes, trying to avoid the nesting, and inferstats has some stats we don't want for batch.